src/types.d.ts

```diff
@@ -5,3 +5,9 @@ declare namespace Express {
         requester: Requester
```

Comment:
Having a types definition file in the source seems... Weird, do you think we could define it elsewhere ?